theseus_gui/src/pages/instance/Index.vue

```diff
@@ -10,9 +10,15 @@
           </span>
         </div>
         <span class="button-group">
-          <Button color="primary" class="instance-button" @click="run($route.params.id)">
-            <PlayIcon />
-            Play
+          <Button
+            :color="instance.installed ? 'primary' : ''"
+            class="instance-button"
+            :disabled="!instance.installed"
+            @click="run($route.params.id)"
+          >
+            <PlayIcon v-if="instance.installed" />
+            <AnimatedLogo v-else class="loading-icon" />
+            {{ instance.installed ? 'Play' : 'Installing' }}
           </Button>
```

Comment:
this shouldn't be a button if it's loading as it shouldn't play the instance if it's still installing